R/environment.r

```diff
@@ -11,10 +11,36 @@ get_shadowenv <- function() {
 }
 
 # Adds functions which do not need any access to the executer into the users searchpath
+#' @importFrom utils getFromNamespace
 init_shadowenv <- function() {
-    # add the accessors to the shadow env itself, so they are actually accessable 
+    # add the accessors to the shadow env itself, so they are actually accessable
     # from everywhere...
     add_to_user_searchpath('.irk.get_shadowenv', get_shadowenv)
     add_to_user_searchpath('.irk.add_to_user_searchpath', add_to_user_searchpath)
 
+    # For the rest of the functions, please explain why the workaround is needed
+    # (=the problem) and link to the issue describing the problem.
+
+    # workaround for problems with vignette(xxx) not bringing up the vignette
+    # content in the browser: https://github.com/IRkernel/IRkernel/issues/267
+    add_to_user_searchpath('print.vignette', function(...){
```

Comment:
I switched to rstudio to lint the code and it doesn't show me this. Do we really want to waste time and effort on something which the RStudio inter doesn not catch?